File: src/1295/solution.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int counter = 0;
        int curr_num_of_digits;
        int curr_num;
        for(int num: nums){
            curr_num_of_digits = 0;
            curr_num = num;
            while(curr_num != 0){
                curr_num /= 10;
                curr_num_of_digits += 1;
            }
            if(curr_num_of_digits % 2 == 0){
                counter += 1;
            }
        }
        return counter;
    }
};

File: src/1295/solution.java
class Solution {
    public int findNumbers(int[] nums) {
        int counter = 0;
        for(int i = 0; i < nums.length; i++){
            if(Integer.toString(nums[i]).length() %2 == 0){
                counter++;
            }
        }       
        return counter;
    }

    public static void main(String[] args) {
        Solution s = new Solution();
        int[] nums = {12,345,2,6,7896};
        System.out.println(s.findNumbers(nums));
    }
}